xblockmufi/private/edit.less

```diff
@@ -1,3 +1 @@
-.xblockmufi_edit {
-    // TODO: Style editor here
-}
+
```

Comment:
If you aren't using `edit.less`, you should remove all mention of it (check `Gruntfile.js`).